src/sentry/metrics/base.py

```diff
@@ -78,3 +78,18 @@ def distribution(
         stacklevel: int = 0,
     ) -> None:
         raise NotImplementedError
+
+    def event(
```

Comment:
No-one is calling this particular function right now. We should do it in this PR to eliminate the chance of having dead-code in the project

Reply:
We are planning to use it in the [ops](https://github.com/getsentry/sentry/blob/699ce0cd1281cc3c7275d0a474a595375c769ae8/src/sentry/processing/backpressure/health.py) repo actually.

Reply:
We are not planning to use it in the ops repo. The ops repo does not access getsentry.
We are planning to use it in the backpressure system in sentry: https://github.com/getsentry/sentry/blob/master/src/sentry/processing/backpressure/health.py

@anonrig , that will happen in the PR @nirajdewani will write right after this one. So no issue.